### ISSUE TITLE
No contact suggestion anymore when writing mails

### ISSUE THREAD

**Issue by annando** [friendica/friendica#7594]
### Expected behavior
When writing a mail via /message/new the "to:" header is empty. When typing in a name, the system should suggest a fitting contact.

### Actual behavior
Nothing happens.

### Steps to reproduce the problem
Write a mail.

### Friendica version you encountered the problem
Current RC

**Comment by AlfredSK** [comment]
When I go to /message:
`TypeError: $(...).name_autocomplete is not a function message:98:14`

When I click on +:
`TypeError: $(...).name_autocomplete is not a function new:98:14`

Details:
```
new:98 Uncaught TypeError: $(...).name_autocomplete is not a function
    at HTMLDocument.<anonymous> (new:98)
    at i (jquery.min.js:2)
    at Object.fireWith [as resolveWith] (jquery.min.js:2)
    at Function.ready (jquery.min.js:2)
    at HTMLDocument.J (jquery.min.js:2)
(anonymous) @ new:98
i @ jquery.min.js:2
fireWith @ jquery.min.js:2
ready @ jquery.min.js:2
J @ jquery.min.js:2
```

**Comment by MrPetovan** [comment]
Looks like I removed something too fast in #7561

**Comment by MrPetovan** [comment]
Closed via #7595
